### PR TITLE
python 3.11 is updating and causing error

### PR DESCRIPTION
**Pull request by changhiskhan**

### CODE REVIEW

File: .github/workflows/cpp.yml [hunk 1]
```diff
@@ -43,10 +43,19 @@ jobs:
...
-          brew pin python@3.10
           cd $(brew --repository)
           git fetch origin --tags
           git checkout 3.6.16  # Arrow 10.0
+          rm -rf /usr/local/bin/2to3
+          rm -rf /usr/local/bin/2to3-3.11        
+          rm -rf /usr/local/bin/idle3          
+          rm -rf /usr/local/bin/idle3.11          
+          rm -rf /usr/local/bin/pydoc3
+          rm -rf /usr/local/bin/pydoc3.11
```
Comment (eddyxu) [inline .github/workflows/cpp.yml:54]: This seems a lot, can `brew link --force` do ?

**Review by eddyxu — COMMENTED** [review]
qq

File: .github/workflows/cpp.yml (hunk 1 above)
Comment (changhiskhan) [reply .github/workflows/cpp.yml:54]: I think this was the hint in the error which I tried first. Not sure why it didn't work.

**Review by eddyxu — APPROVED** [review]
kk. lets merge this for now. not sure how long this fix can last though.

**Pull request merged**